contracts/core/OTCWrapper.sol

```diff
@@ -0,0 +1,691 @@
+        uint256 _min,
+        uint256 _max
+    ) external onlyOwner {
+        require(_underlying != address(0), "OTCWrapper: asset address cannot be 0");
+        require(_min > 0, "OTCWrapper: minimum notional cannot be 0");
+        require(_max > 0, "OTCWrapper: maximum notional cannot be 0");
+
+        minMaxNotional[_underlying] = MinMaxNotional(_min, _max);
+    }
+
+    /**
+     * @notice sets the whitelist status of market maker addresses
+     * @dev can only be called by owner
+     * @param _marketMakerAddress address of the market maker
+     * @param _isWhitelisted bool with whitelist status
+     */
+    function setWhitelistMarketMaker(address _marketMakerAddress, bool _isWhitelisted) external onlyOwner {
+        require(_marketMakerAddress != address(0), "OTCWrapper: market maker address cannot be 0");
+
+        isWhitelistedMarketMaker[_marketMakerAddress] = _isWhitelisted;
+    }
+
+    /**
+     * @notice sets the fee for a given the underlying asset
+     * @dev can only be called by owner
+     * @param _underlying underlying asset address
+     * @param _fee fee amount in bps (4bps = 0.04%)
+     */
+    function setFee(address _underlying, uint256 _fee) external onlyOwner {
+        require(_underlying != address(0), "OTCWrapper: asset address cannot be 0");
+
+        fee[_underlying] = _fee;
+    }
+
+    /**
+     * @notice sets the beneficiary address
+     * @dev can only be called by owner
+     * @param _beneficiary beneficiary address
+     */
+    function setBeneficiary(address _beneficiary) external onlyOwner {
+        require(_beneficiary != address(0), "OTCWrapper: beneficiary address cannot be 0");
+
+        beneficiary = _beneficiary;
+    }
+
+    /**
+     * @notice sets the fill deadline duration
+     * @dev can only be called by owner
+     * @param _fillDeadline fill deadline duration
+     */
+    function setFillDeadline(uint256 _fillDeadline) external onlyOwner {
+        require(_fillDeadline > 0, "OTCWrapper: fill deadline cannot be 0");
+
+        fillDeadline = _fillDeadline;
+    }
+
+    /************************************************
+     *  DEPOSIT & WITHDRAWALS
+     ***********************************************/
+
+    /**
+     * @notice allows market maker to deposit collateral
+     * @dev can only be called by the market maker who is the order seller
+     * @param _orderID id of the order
+     * @param _amount amount to deposit
+     */
+    function depositCollateral(uint256 _orderID, uint256 _amount) external nonReentrant {
+        require(orderStatus[_orderID] == OrderStatus.Succeeded, "OTCWrapper: inexistent or unsuccessful order");
+
+        Order memory order = orders[_orderID];
+
+        require(order.seller == _msgSender(), "OTCWrapper: sender is not the order seller");
+
+        IERC20 collateralInterface = IERC20(order.collateral);
+
+        // market maker inflow - an approve() or permit() by the msg.sender is required beforehand
```

Comment:
still use msg.sender in comments -> for consistency can we move to _msgSender()?

Reply:
https://github.com/ribbon-finance/GammaProtocol-OTC/pull/2/commits/ee82715ad0565ee04d595d69a84a42839e7f65b1